sambacc/commands/dcmain.py

```diff
@@ -41,9 +41,10 @@ def main(args: typing.Optional[typing.Sequence[str]] = None) -> None:
         raise Fail("missing container identity")
 
     pre_action(cli)
-    skip = action_filter(cli)
+    ctx = CommandContext(cli)
+    skip = skips.test(ctx)
     if skip:
-        print(f"Action skipped: {skip}")
+        print(f"Command Skipped: {skip}")
```

Comment:
Should probably be "Command skipped" (lower case on 2nd word)

Reply:
I intentionally wanted that one to stand out and use title case.